cmd/create/quick-start-cluster/cmd.go

```diff
@@ -0,0 +1,222 @@
+package quickstartcluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	//"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+
+	//"github.com/spf13/pflag"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	cfTypes "github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+
+	// "github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/output"
+    "github.com/sirupsen/logrus"
+)
+
+var Cmd = makeCmd()
+
+func makeCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "quick-start-cluster",
+		Short: "Create quick start cluster",
+		Long:  "Create quick start hcp cluster.",
+		Example: `  # Create a quick-start-cluster
+  rosa create quick-start-cluster`,
+		Run:  run,
+		Args: cobra.NoArgs,
+	}
+}
+
+var args struct {
+    namePrefix string
+    vpcCidr string
+    availabilityZonesCount int
+    tags map[string]string
+}
+
+func init() {
+    flags := Cmd.Flags()
+    flags.StringVar(
+		&args.namePrefix,
+		"name",
+		"dmoskale-example",
+		"Prefix for naming resources",
+	)
+    flags.StringVar(
+        &args.vpcCidr,
+        "vpc-cidr",
+        "10.0.0.0/16",
+        "CIDR block for the VPC",
+    )
+    flags.IntVar(
+        &args.availabilityZonesCount,
+        "availability-zones-count",
+        2,
+        "Number of availability zones",
+    )
+    flags.StringToStringVar(
+        &args.tags,
+        "tags",
+        nil,
+        "Additional tags",
+    )
+    output.AddFlag(Cmd)
+}
+
+
+func run(cmd *cobra.Command, _ []string) {
+    r := rosa.NewRuntime().WithAWS().WithOCM()
+    defer r.Cleanup()
+
+    err := createVPCStack("cmd/create/quick-start-cluster/vpc.yaml", args.namePrefix, args.vpcCidr, args.availabilityZonesCount, args.tags)
+    if err != nil {
+        r.Reporter.Errorf(err.Error())
+        os.Exit(1)
+    }
+
+    r.Reporter.Infof("VPC created")
+    r.Reporter.Infof("To view all created resource stacks, run `aws cloudformation list-stacks --stack-status-filter CREATE_COMPLETE`")
```

Comment:
it would be good if you can get the outputs of the required resources needed to create a cluster with